gatsby-directus-website/index.md

```diff
@@ -0,0 +1,290 @@
+---
+title: Build a website with Directus and Gatsby
+description: Let's learn how to build a simple website using Directus as a CMS. We'll use the official plugin to create pages dynamically based on a Directus collection and optimize images.
+author:
+  avatar_file_name: alvin-bryan.jpeg
+  name: Alvin Bryan
+---
+
+## Introduction
+
+Hello! This is a blog post to show you how to create a simple static website using Gatsby and Directus as a headless CMS.
+
+We'll make use of the official Directus Gatsby plugin whichs lets hook into the best feature of Gatsby, which is image compression.
+
+## Before You Start
+
+You will need:
+
+- Node.js and a code editor installed.
+- A Directus project - [follow the quickstart guide](https://docs.directus.io/getting-started/quickstart) to create one.
+- Knowledge of Gatsby and React.
+
+## Setup
+
+Create a new gatsby project using NPM:
+
+```sh
+npm init gatsby
+```
+
+Respond with the following:
+
+```sh
+What would you like to call your site?
+✔ · Directus Gatsby Blog
+
+What would you like to name the folder where your site will be created?
+✔ code/ directus-gatsby-blog
+
+✔ Will you be using JavaScript or TypeScript?
+· JavaScript
+
+✔ Will you be using a CMS?
+· No (or I'll add it later)
+
+✔ Would you like to install a styling system?
+· No (or I'll add it later)
+
+? Would you like to install additional features with other plugins?
+· Done
+
+❯◯ Add the Google gtag script for e.g. Google Analytics
+· Done
+
+Shall we do this? (Y/n)
+› Yes
+```
```

Comment:
````suggestion
Create a new Gatsby project on your terminal and walk through the wizard:

```sh
npm init gatsby

What would you like to call your site?
  › Directus Gatsby Blog
What would you like to name the folder where your site will be created?
  › directus-gatsby-blog
Will you be using JavaScript or TypeScript?
  › JavaScript
Will you be using a CMS?
  › No (or I'll add it later)
Would you like to install a styling system?
  › No (or I'll add it later)
Would you like to install additional features with other plugins?
  › Done
Add the Google gtag script for e.g. Google Analytics
  › Done
Shall we do this? (Y/n)
  › Yes
```
````

Shrinking it down visually